### PR TITLE
Add Fix Description for DCJS.

### PR DESCRIPTION
**Pull request by shmao**
@joshfree

### CODE REVIEW

File: releases/UWP/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ When using Visual Studio this package requires Visual Studio 2017 or later.
 
 ### 5.3.1 (March 7, 2017)
 
-- Fixes [DataContractJsonSerializer failing in Microsoft.NetCore.UniversalWindowsPlatform 5.3.0](https://github.com/dotnet/corefx/issues/10374)
+- Fix a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer fail to serialize any [DataContract] type whose default constructor is not public.
```
Comment (joshfree) [inline releases/UWP/README.md]: > Fix a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer fail to serialize 

I would change this to

Fixes a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer to fail to serialize

Comment (joshfree) [inline releases/UWP/README.md]: otherwise LGTM

File: releases/UWP/README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ When using Visual Studio this package requires Visual Studio 2017 or later.
 
 ### 5.2.3 (March 7, 2017)
 
-- Fixes [DataContractJsonSerializer failing in Microsoft.NetCore.UniversalWindowsPlatform 5.2.2](https://github.com/dotnet/corefx/issues/10374)
+- Fix a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer fail to serialize any [DataContract] type whose default constructor is not public.
```
Comment (joshfree) [inline releases/UWP/README.md]: same change here as well.

**Comment by shmao** [comment]
@joshfree Thanks for reviewing the changes. I've updated the PR per your suggestion.

**Pull request merged**